fetch.bs

```diff
@@ -4210,12 +4264,23 @@ steps:
   then:
 
   <ol>
+   <li>
+    <p>Let <var>processEndOfBody</var> given <var>nullBytesOrFailure</var> be the following
+    steps:
+
+    <ol>
+     <li><p>Run <var>processResponseDone</var>.
```

Comment:
Why is this needed? Wouldn't this run as part of reading the body?

Reply:
I was trying to fiddle with the order of things, but will revise to make it cleaner.